lib/manager/mix/update.js

```diff
@@ -0,0 +1,32 @@
+function updateDependency(currentFileContent, upgrade) {
+  try {
+    logger.debug(`mix.updateDependency: ${upgrade.newValue}`);
+
+    if (upgrade.depType !== 'hex')
```

Comment:
If we don't support updating non-hex depTypes then they should all be given a `skipReason` during extraction phase

Reply:
This still applies. We should not be throwing an error here if we let it go too far.